extensions/positron-python/resources/walkthrough/bundledExtensions.md

```diff
@@ -0,0 +1,5 @@
+Positron already includes several extensions to support Python development, such as Jupyter for notebook support, Ruff for linting and formatting, and Quarto for publishing. 
+These are installed automatically the first time you launch Positron.
+
+In addition to these, Positron uses built-in extensions to power core features like the Python backend that connects your code to the IDE interface. 
```

Comment:
I'm not sure how much people care about this part, but it seems important to mention that Positron already comes with everything to run Python